addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -23,11 +23,11 @@
                             Discuss
                         </div>
                     </t>
-                    <t t-if="threadViewTopbar.thread and threadViewTopbar.thread.isChannelDescriptionChangeable">
+                    <t t-if="threadViewTopbar.hasDescription">
```

Comment:
Code is hard to understand because field is badly named. This part should be shown even if the thread doesn't have a description, because the goal is also to display the placeholder for adding one.